iOS/MMIWG2S.xcodeproj/project.pbxproj

```diff
@@ -7,63 +7,69 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		22DA2EEC25FAD3DB00AB601E /* README.md in Resources */ = {isa = PBXBuildFile; fileRef = 22DA2EEB25FAD3D900AB601E /* README.md */; };
```

Comment:
The `README` shouldn't be bundled in the app. It just needs to live in the directory.